.github/workflows/test-application.yaml

```diff
@@ -51,3 +52,138 @@ jobs:
               run: curl "https://cs.symfony.com/download/php-cs-fixer-v2.phar" --output ${{ steps.php-cs-fixer-path.outputs.path }} && chmod +x ${{ steps.php-cs-fixer-path.outputs.path }}
             - name: Run php-cs-fixer
               run: ./${{ steps.php-cs-fixer-path.outputs.path }} fix --dry-run --diff
+
+    test:
+        name: "Run tests with php ${{ matrix.php-version }} (${{ matrix.phpcr-transport }})"
```

Comment:
would be nice to also add the database name e.g. mysql / postgresql

Reply:
should be done in `release/1.6` then ... I can create another pr there

Reply:
I guess so... Create a PR there, then we can also see how back merging works :see_no_evil: